index.html

```diff
@@ -60,9 +60,162 @@ <h3>I can help you build a product , feature or website Look through some of my
+
+         <ul class="lan1">
+             <li class="l1"> <a href="#"> <img src="image/css.png" alt=""></a> </li>
+             <li class="l1"> <a href="#"> <img src="image/html.png" alt=""></a> </li>
+             <li class="l1"> <a href="#"> <img src="image/bootstrap.png" alt=""></a> </li> 
+             <li class="l1"> <a href="#"> <img src="image/runy.png" alt=""></a> </li> 
+ 
+         </ul>
+    
+         <button class="btn1"> See Project</button></div>
+
+        
+        
+          
+        
+            <div class="project2" >
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+          <button class="btn2">See Project </button> </div>
+
+          <div class="project2" id="project22">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+          <button class="btn2">See Project </button> </div>
+
+          <div class="project2" id="project21">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+         <button class="btn2">See Project </button> </div>
+
+
+          <div class="project2" id="project20">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+         <button class="btn2">See Project </button> </div>
+        
+                     
+        </section>
+           
+
+
+
+       <section class="body3">
+        <h1 class="s1h11">About me</h1>
```

Comment:
- [ OPTIONAL] You should have only one `<h1>` tag in HTML document for semantic reasons. You have already used one in the headline section. You may use `<h2>` instead.